src/lib/translations/index.js

```diff
@@ -15,125 +27,15 @@ export const config = {
-        {
-            locale: 'nn',
-            key: 'membership',
-            routes: ['/membership', '/membership/registrationComplete'],
-            loader: async () => (await import('../../content/translations/nn/membership.json')).default,
-        },
-        {
-            locale: 'en',
-            key: 'volunteer',
-            routes: ['/volunteer'],
-            loader: async () => (await import('../../content/translations/en/volunteer.json')).default,
-        },
-        {
-            locale: 'nb',
-            key: 'volunteer',
-            routes: ['/volunteer'],
-            loader: async () => (await import('../../content/translations/nb/volunteer.json')).default,
-        },
-        {
-            locale: 'nn',
-            key: 'volunteer',
-            routes: ['/volunteer'],
-            loader: async () => (await import('../../content/translations/nn/volunteer.json')).default,
-        },
-        {
-            locale: 'en',
-            key: 'events',
-            routes: ['/events'],
-            loader: async () => (await import('../../content/translations/en/events.json')).default,
-        },
-        {
-            locale: 'nb',
-            key: 'events',
-            routes: ['/events'],
-            loader: async () => (await import('../../content/translations/nb/events.json')).default,
-        },
-        {
-            locale: 'nn',
-            key: 'events',
-            routes: ['/events'],
-            loader: async () => (await import('../../content/translations/nn/events.json')).default,
-        },
-        {
-            locale: 'en',
-            key: 'email',
-            loader: async () => (await import('../../content/translations/en/email.json')).default,
-        },
-        {
-            locale: 'nb',
-            key: 'email',
-            loader: async () => (await import('../../content/translations/nb/email.json')).default,
-        },
-        {
-            locale: 'nn',
-            key: 'email',
-            loader: async () => (await import('../../content/translations/nn/email.json')).default,
-        },
-        {
-            locale: 'en',
-            key: 'counter',
-            loader: async () => (await import('../../content/translations/en/counter.json')).default,
-        },
-        {
-            locale: 'nb',
-            key: 'counter',
-            loader: async () => (await import('../../content/translations/nb/counter.json')).default,
-        },
-        {
-            locale: 'nn',
-            key: 'counter',
-            loader: async () => (await import('../../content/translations/nn/counter.json')).default,
-        },
-    ],
+    loaders: Object.entries(KEYS).flatMap(([key, routes]) =>
+        LOCALES.map((locale) => ({
+            locale,
+            key,
+            ...(routes ? { routes } : {}),
+            loader: async () =>
+                (await import(`../../content/translations/${locale}/${key}.json`)).default,
+        })),
+    ),
```

Comment:
Very nice! Every time I added a translation file I thought "I need to fix this", and summarily forgot 👌

Reply:
:)